### PR TITLE
BREAKING CHANGE: getFee returns promise, express fee in XRP in instru…

### PR DESCRIPTION
**Pull request by clark800**
…ctions response, use rawRequest to start decoupling Remote from API

### CODE REVIEW

File: src/api/common/connection.js [hunk 1]
```diff
@@ -62,6 +73,10 @@ class Connection extends EventEmitter {
         }
         this.emit(data.id.toString(), data);
       } else if (isStreamMessageType(data.type)) {
+        if (data.type === 'ledgerClosed') {
```
Comment (darkdarkdragon) [inline src/api/common/connection.js]: I think this should be `ledger_closed`, and in `isStreamMessageType` too.

Comment (clark800) [inline src/api/common/connection.js]: No, ripple-lib/core renames it from ledgerClosed to ledger_closed

https://ripple.com/build/rippled-apis/#subscribe
https://github.com/ripple/ripple-lib/blob/develop/src/core/remote.js#L503

Comment (darkdarkdragon) [inline src/api/common/connection.js]: hm. unexpected :smile:

**Comment by darkdarkdragon** [comment]
LGTM

**Pull request merged**